### PR TITLE
[Feat]: whiteboard action buttons

### PR DESCRIPTION
**Pull request by CulturalProfessor**
Fixes(#13 ) v1

### CODE REVIEW

**Comment by CulturalProfessor** [comment]
image

File: whiteboard/i18n/en.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "create_whiteboard":"▢ Whiteboard"
```
Comment (shiqimei) [inline whiteboard/i18n/en.json:2]: use the value "Create Whiteboard"

File: whiteboard/handlers/ExecuteActionButtonHandler.ts [hunk 2]
```diff
@@ -27,11 +32,59 @@ export class ExecuteActionButtonHandler {
         try {
             const { actionId, user } = data;
             switch (actionId) {
-                case UtilityEnum.RENAME_BUTTON_ACTION_ID:
-                    console.log("RENAME BUTTON CLICKED");
-                    return context.getInteractionResponder().successResponse();
                 case UtilityEnum.CREATE_WHITEBOARD_MESSAGE_BOX_ACTION_ID:
-                    console.log("CREATE NEW WHITEBOARD BUTTON CLICKED");
+                    console.log("BUTTON CLICKED");
```
Comment (shiqimei) [inline whiteboard/handlers/ExecuteActionButtonHandler.ts:36]: remove unused logs

**Review by shiqimei — CHANGES_REQUESTED** [review]

**Comment by CulturalProfessor** [comment]
image

**Comment by shiqimei** [comment]
LGTM! @CulturalProfessor Thanks for you great work ;)

**Pull request merged**